src/mono/sample/wasm/browser-bench/appstart-frame.html

```diff
@@ -12,7 +12,6 @@
   <link rel="preload" href="./mono-config.json" as="fetch" crossorigin="anonymous">
   <link rel="prefetch" href="./dotnet.wasm" as="fetch" crossorigin="anonymous">
   <!-- users should consider if they optimize for the first load or subsequent load from memory snapshot -->
-  <link rel="prefetch" href="./icudt.dat" as="fetch" crossorigin="anonymous">
```

Comment:
Do we want to drop it even from advanced sample or just here?

Reply:
I will add it to some other PR.